src/liana/method/sc/_compute_global_score.py

```diff
@@ -0,0 +1,217 @@
+import numpy as np
```

Comment:
I moved this one to sp; perhaps we need a more specific name for this one also; compute_global_specificity perhasps?